backend/dataall/modules/mlstudio/cdk/mlstudio_extension.py

```diff
@@ -199,22 +206,3 @@ def extent(setup: EnvironmentSetup):
             parameter_name=f'/{_environment.resourcePrefix}/{_environment.environmentUri}/sagemaker/sagemakerstudio/domain_id',
         )
         return sagemaker_domain
-
```

Comment:
This is great! We can get rid of this legacy piece of code. Basically data.all will always create a SageMaker domain